pkg/i2gw/notifications/notifications.go

```diff
@@ -69,8 +71,10 @@ func (na *NotificationAggregator) CreateNotificationTables() {
 			t.AppendRow(table.Row{n.Type, n.Message, convertObjectsToStr(n.CallingObjects)})
 		}
 
-		t.Render()
+		tables = append(tables, t)
```

Comment:
would it be simpler to have tables as list of strings. 

and then you do `tables = append(tabels,t.render())` ?

Then you would not need to render them outside this function
see the related comment on print.go

Reply:
But when we apply t.Render() it automatically outputs it on the terminal. It returns a string as well. So there is no point of having just t.Render()